types/mui-datatables/mui-datatables-tests.tsx

```diff
@@ -157,15 +175,15 @@ const MuiCustomTable: React.FC<Props> = props => {
         },
     };
 
-    return <MUIDataTable title={props.title} data={data} columns={columns} options={TableOptions} />;
+    return <MUIDataTable title={props.title} data={data} columns={columns} options={TableOptions}/>;
 };
 
 const TableFruits = [
-    { id: 1, name: 'Apple', color: 'Red', amount: 1 },
-    { id: 2, name: 'Pear', color: 'Green', amount: 2 },
-    { id: 3, name: 'Strawberry', color: 'Red', amount: 5 },
-    { id: 4, name: 'Banana', color: 'Yellow', amount: 7 },
-    { id: 5, name: 'Orange', color: 'Orange', amount: 9 },
+    {id: 1, name: 'Apple', color: 'Red', amount: 1},
+    {id: 2, name: 'Pear', color: 'Green', amount: 2},
+    {id: 3, name: 'Strawberry', color: 'Red', amount: 5},
+    {id: 4, name: 'Banana', color: 'Yellow', amount: 7},
+    {id: 5, name: 'Orange', color: 'Orange', amount: 9},
```

Comment:
```suggestion
    { id: 1, name: 'Apple', color: 'Red', amount: 1 },
    { id: 2, name: 'Pear', color: 'Green', amount: 2 },
    { id: 3, name: 'Strawberry', color: 'Red', amount: 5 },
    { id: 4, name: 'Banana', color: 'Yellow', amount: 7 },
    { id: 5, name: 'Orange', color: 'Orange', amount: 9 },
```

Reverted autoformatting